### PR TITLE
use makefile for setup - fixes #29

### LINKED ISSUE #29: Use Makefile and make instead of package.json and npm
Currently, README includes instructions that use npm to execute docker related commands Eg `npm run docker:up` for `docker-compose -f ../../docker-compose.dev.yml up` etc...  Think it's more logical to use Makefile for that, looks nicer and no need to install nodejs.

Example https://github.com/ViderumGlobal/ckanext-querytool/blob/master/Makefile

## Tasks

* [ ] Create Makefile with a similar command that is in package.json Eg: `make docker:up`
* [ ] Update README

### PR DESCRIPTION
**Pull request by zelima**
Plus README updated appropriately

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+start:
+	docker-compose -f ../../docker-compose.dev.yml up
+
+shell:
+	docker-compose -f ../../docker-compose.dev.yml exec ckan-dev bash
+
+test:
+	docker-compose -f ../../docker-compose.dev.yml exec ckan-dev nosetests --ckan --nologcapture --reset-db -s -v --with-pylons=/srv/app/src_extensions/ckanext-ed/test.ini /srv/app/src_extensions/ckanext-ed/ ${ARGS}
+
+reset:
```
Comment (roll) [inline Makefile]: my 2c
I would not add this because it's too dangerous. 
I mean if you really know what are you doing you can use the full command.

Comment (amercader) [reply Makefile]: Yes, that's a command that you shouldn't need to run very often. I don't think it need to be in the Makefile

Comment (zelima) [reply Makefile]: :+1:

**Comment by zelima** [comment]
@amercader only command related to E2E tests and Gulp. So leaving for now

---

 We write and store acceptance E2E tests inside the top-level `tests` directory. For running it we also have to switch to the test server:

```bash
$ make start
```

In another terminal window run the test command:

```bash
$ npm install nightwatch --no-save # Install test runner
$ npm run test:e2e
```

See the `how to write E2E tests` guide:
- http://nightwatchjs.org/guide

### Working with static files

Put your scripts/fonts/etc inside the `ckanext/ed/fanstatic` folder and images inside the `ckanext/ed/public` folder. It can be used as usual ckan `fanstatic` and `public` contents.

At the same time, we use CSS preprocessor (LESS) to build our styles. Put your styles inside the `ckanext/ed/less` and build it:

```bash
$ npm run static:build # Option 1: one-time build
$ npm run static:watch # Option 2: build on every change
```

Processed styles will be put to the `ckanext/ed/fanstatic/css` folder.

**Pull request merged**